operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/AbstractConfigurationService.java

```diff
@@ -60,12 +54,14 @@ protected void throwExceptionOnNameCollision(
     final var key = keyFor(controller);
     final var configuration = configurations.get(key);
     if (configuration == null) {
-      log.warn(
-          "Configuration for controller '{}' was not found. {}", key, getControllersNameMessage());
+      logWarning(key, getControllersNameMessage());
```

Comment:
wonder if we need to log things here or should be a responsibility of the caller

Reply:
I added that logging feature a while back because missing controller configuration could lead to rather unclear subsequent error messages. It's definitely not needed but it helps the user figure out what might be wrong.